.github/workflows/openapi-validate.yml

```
@@ -0,0 +1,89 @@
+name: Validate OpenAPI Specification
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  validate-openapi:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)